### ISSUE TITLE
Check with all the device implemetation

### ISSUE THREAD

**Issue by pushkarkadam** [pushkarkadam/handpose#27]

**Comment by pushkarkadam** [comment]
So far implemented to device as follows:

- CNN model transfer function
- Data tensor in train function
- All the losses in the total loss

**Comment by pushkarkadam** [comment]
Turns out each and every value that is being computed with a tensor should be connected to device.

```python
import torch

DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")

A = torch.Tensor([[1,2],[3,4]]).to(DEVICE)
b = torch.tensor(2).to(DEVICE)

B = A / b
```
